packages/@aws-cdk/integ-tests-alpha/lib/assertions/api-call-base.ts

```diff
@@ -125,6 +125,7 @@ export abstract class ApiCallBase extends Construct implements IApiCall {
   protected expectedResult?: string;
   protected flattenResponse: string = 'false';
   protected stateMachineArn?: string;
+  protected outputPaths: string[] = [];
```

Comment:
Changes related to outputPaths were to fix a bug found where using getAtt and getAttString were not updating the outputPaths. This was an unrelated fix that I decided to address as part of this PR since it was impacting the aws-autoscaling-hooktargets integ test called integ.queue-hook